.circleci/build-assets.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -ex
+
+echo "Begin build for $CIRCLE_ENV${PR_ENV:+ for }$PR_ENV. Pantheon test environment is $TERMINUS_SITE.$TERMINUS_ENV"
+
+# Build assets with Composer
+composer -n build-assets
+
+# Compile Sass or run any other build steps necessary
```

Comment:
I think we should have a section in the README with instructions on how to do so.

Alternatively we could move [the child theme from my other repo](https://github.com/ataylorme/Advanced-WordPress-on-Pantheon/tree/master/web/wp-content/themes/twentyseventeen-child), which compiles Sass via gulp, into this repo as an example.